### DISCUSSION TITLE
How to use custom css

### DISCUSSION THREAD

**Discussion by Nzii3** [retypeapp/retype#469] [Q&A] [answered]
How do I use the custom css and where do I put it in the directory? What I mean by how do I use it, are there any docs on what each [component](https://retype.com/components/) is related to in HTML/CSS so I can change the colors and stuff?

**Comment by geoffreymcgill** [comment, marked as answer]
The following post should help, see

https://github.com/retypeapp/retype/discussions/282#discussioncomment-2343005

Any custom CSS or HTML is outside of the scope of support for Retype and is up to the author to create and maintain. 

Hope this helps.

**Reply by Nzii3** [reply]
There is documentation on how to apply a custom class for a container, how would I do that for a button?
